packages/network-controller/src/NetworkController.test.ts

```diff
@@ -2095,128 +1181,6 @@ describe('NetworkController', () => {
           expect(fakeMetamaskProviders[0].stop).toHaveBeenCalled();
         });
       });
-
-      it('updates the version of the current network in state (assuming that the request for net_version is made successfully)', async () => {
```

Comment:
I am not sure whether this test was removed — the diff is a bit funky here — but we should keep this test if we can (renaming the test and adjusting the expectation).